packages/ragbits-document-search/src/ragbits/document_search/retrieval/rephrasers/base.py

```diff
@@ -1,20 +1,37 @@
 import abc
+from typing import Optional
+
+from ragbits.core.llms.base import LLMOptions
 
 
 class QueryRephraser(abc.ABC):
     """
     Rephrases a query. Can provide multiple rephrased queries from one sentence / question.
     """
 
-    @staticmethod
     @abc.abstractmethod
-    def rephrase(query: str) -> list[str]:
+    async def rephrase(self, query: str, options: Optional[LLMOptions] = None) -> list[str]:
         """
         Rephrase a query.
 
         Args:
             query: The query to rephrase.
+            options: OptionaL options to fine-tune the rephraser behavior.
```

Comment:
```suggestion
            options: Optional configuration of the the rephraser behavior.
```

(it's very nit-picky but fixes the "L" in "OptionaL", the awkward sounding "Optional options", and gets rid of the world "fine-tuning" which got my confused at first since usually it has a different  meaning when used with LLMs)

Reply:
thank you, fixed